### DISCUSSION TITLE
How to achieve galaxy-brain?

### DISCUSSION THREAD

**Discussion by pseudocc** [real-life-copium/yolo#4] [Q&A] [answered]
How to achieve galaxy-brain?

**Comment by main76** [comment, marked as answer]
I don't know either!

**Reply by ismailcanvarli** [reply]
hello

**Reply by pseudocc** [reply]
Hi there.

**Reply by ismailcanvarli** [reply]
ı am trying to galaxy brain

**Reply by pseudocc** [reply]
Hi @ismailcv, you could reply this two Q&A discussions(#8 & #9), and I will mark your comments as answers, then you will get the badge.
